### ISSUE TITLE
Locally test and debug an ASP.NET website in a Docker container

### ISSUE THREAD

**Issue by carolynvs** [getcarina/getcarina.com#639]
It's pretty easy when developing on Windows to reference things that won't work on linux with core clr. Knowing how to test and debug on a local container should help a ton, and build up a good foundation for the next part: ci/cd.

Split off from #6

**Comment by everett-toews** [comment]
@carolynvs Is this done?

**Comment by carolynvs** [comment]
Nope! It's not even on my radar right now which is why I haven't licked this :cookie: yet.
